### PR TITLE
add missing pkg-config build dep in recent GObject-Introspection, GLib & cairo easyconfigs

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by JackPerdue** [comment]
Are flex/Bison/pkg-config deps or builddeps?

**Comment by JackPerdue** [comment]
[2017b_vers.txt](https://github.com/easybuilders/easybuild-easyconfigs/files/1931390/2017b_vers.txt)
FWIW... what I'm currently targeting for 2017b (in hopes of doing it better for 2018a)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2015.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2884161455195bacb8d7362a595e339c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2537.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b45174edc5016c747e7bf3da700c09f4 for a full test report.

**Comment by vanzod** [comment]
While testing this PR, I got the same error when installing `GLib-2.52.0-intel-2017a.eb`.
Maybe adding those too to this PR?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in this PR)
node2537.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1b224999564ce60509a444d6ae5938a2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 12 out of 12 (12 easyconfigs in this PR)
node2118.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1b0b3de25b9c437b9e8dd722ac4458f8 for a full test report.

**Comment by vanzod** [comment]
`cairo-1.14.8-intel-2017a.eb` is also missing `pkg-config` as build dependency:

```
checking for pkg-config... no
configure: error: pkg-config >=  required but not found (http://pkgconfig.freedesktop.org/)
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 17 out of 17 (17 easyconfigs in this PR)
node2118.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a7706be8f4aba7f11b88dd0dee016bb8 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 15 out of 17 (17 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/5d854b0bd504ad2742a53d8bb1c9253a for a full test report.

File: easybuild/easyconfigs/c/cairo/cairo-1.14.12-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -14,6 +14,8 @@ source_urls = ['http://cairographics.org/releases/']
 sources = [SOURCE_TAR_XZ]
 checksums = ['8c90f00c500b2299c0a323dd9beead2a00353752b2092ead558139bd67f7bf16']
 
+builddependencies = [('pkg-config', '0.29.2')]
```
Comment (vanzod) [inline easybuild/easyconfigs/c/cairo/cairo-1.14.12-GCCcore-6.4.0.eb]: There is already a `builddependencies` array in the easyconfig file. Needs to be merged.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 17 out of 17 (17 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/0d662652e74270cc9c42f64c4f78503b for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Comment by boegel** [comment]
:+1:

**Pull request merged**